client/landing/stepper/declarative-flow/internals/sensei.scss

```diff
@@ -0,0 +1,5 @@
+.sensei.launchpad {
+	.badge {
+		height: auto;
+	}
+}
```

Comment:
Finding the placement of this file a bit odd in framework's "internal" folder. Not a blocker for merging but something to consider.

Reply:
Moved this to launchpad directory in 280af0fde890863542d855924e60ef3ea6c353a3